server.h

```diff
@@ -58,19 +63,31 @@ class CTFOServer final {
  public:
   using Storage = CTFOStorage<SherlockStreamPersister>;
   using MidichloriansServer = current::midichlorians::server::MidichloriansHTTPServer<CTFOServer>;
+  using RESTStorage = RESTfulStorage<Storage,
+#ifdef USE_BASIC_HYPERMEDIA
+                                     current::storage::rest::Hypermedia>;
+#else
+                                     current::storage::rest::AdvancedHypermedia>;
+#endif
 
   explicit CTFOServer(const std::string& cards_file,
                       int port,
                       const std::string& storage_file,
                       int midichlorians_port,
                       const std::string& midichlorians_file,
+                      int rest_port,
+                      const std::string& rest_url_prefix,
                       const std::chrono::milliseconds tick_interval_ms,
                       const bool debug_print_to_stderr = false)
```

Comment:
+1

@grixa , what do you think of `struct CTFOServerParams` (or even a `CURRENT_STRUCT(CTFOServerParams)` to self-report via some `/params`), and a dot-notation for it, ex. `CTFOServer server(CTFOServerParams().SetCardsFile("...").SetCTFOAPIPort(...).Set...(...));` ?